### PR TITLE
Fix prettier errors

### PR DESCRIPTION
**Pull request by jhowlett-scottlogic**

### CODE REVIEW

File: src/log.js [hunk 1]
```diff
@@ -14,144 +14,176 @@ let logLevel = 1;
...
-    verbose("  )\\())           (         )\\    (()/((()/(  (()/(      (    (  (     (   ");
-    verbose(" ((_)\\   `  )    ))\\  (  ((((_)(   /(_))/(_))  /(_)) (   )(   )\\))(   ))\\  ");
-    verbose("   ((_)  /(/(   /((_) )\\ ))\\ _ )\\ (_)) (_))   (_))_| )\\ (()\\ ((_))\\  /((_) ");
-    verbose("  / _ \\ ((_)_\\ (_))  _(_/((_)_\\(_)| _ \\|_ _|  | |_  ((_) ((_) (()(_)(_))   ");
-    verbose(" | (_) || '_ \\)/ -_)| ' \\))/ _ \\  |  _/ | |   | __|/ _ \\| '_|/ _` | / -_)  ");
-    verbose("  \\___/ | .__/ \\___||_||_|/_/ \\_\\ |_|  |___|  |_|  \\___/|_|  \\__, | \\___|  ");
-    verbose("        |_|                                                  |___/         ");
-    verbose("");
-    return;
+  verbose("");
```
Comment (ColinEberhardt) [inline src/log.js:66]: can you eslint ignore this section so that the original formatting can be retained?

**Review by ColinEberhardt — CHANGES_REQUESTED** [review]
one very minor request.

**Comment by ColinEberhardt** [comment]
> yes i missed that, i have added it now

thanks @jhowlett-scottlogic - feel free to merge when you've resolved conflicts

**Comment by jhowlett-scottlogic** [comment]
@ColinEberhardt this needs re-reviewing before it can be merged as you have requested changes

**Review by ColinEberhardt — APPROVED** [review]

**Pull request merged**